chapters/synchronization.txt

```diff
@@ -4855,7 +4855,7 @@ family.
 The pname:srcQueueFamilyIndex parameter of the barrier must: be set to the
 source queue family index, and the pname:dstQueueFamilyIndex parameter to
 the destination queue family index.
-pname:dstStageMask is ignored for such a barrier, such that no visibility
+pname:dstAccessMask is ignored for such a barrier, such that no visibility
```

Comment:
I think it makes it unclear what is ignored. Is the mask still considered wrt to VUs?

Elswhere it says:
> [...] and the first access scope includes no access, as if `srcAccessMask` was `0`.